structs.go

```diff
@@ -324,6 +353,72 @@ type ChannelFollow struct {
 	WebhookID string `json:"webhook_id"`
 }
 
+// ThreadMetadata holds information about a thread.
+type ThreadMetadata struct {
+	// Whether the channel is archived.
+	Archived bool `json:"archived"`
+
+	// The period of inactivity allowed before the channel automatically archives.
+	AutoArchiveDuration ArchiveDuration `json:"auto_archive_duration"`
+
+	// A timestamp of when the thread's archive status was last changed.
+	ArchiveTimestamp Timestamp `json:"archive_timestamp"`
+
+	// Whether the thread is locked.
+	Locked bool `json:"locked"`
+}
+
+// ArchiveDuration represents the increments of time at which a thread auto-archives, in seconds.
```

Comment:
Is this seconds or hours? The comment and the definition of `ArchiveDurationOneHour` are conflicting.

Reply:
That was meant to be minutes, I believe I made that comment before realizing it was minutes instead of seconds.